src/components/SignInModal.js

```diff
@@ -0,0 +1,46 @@
+import { CancelSignInIcon } from '../icons'
+import React from 'react'
+import '../index.css'
+
+
+const SignInModal = ({setShowSignInModal, setShowSignUpModal}) => {
+  return (
+    <>
+    <div className='signInModal'>
+        
+        <div className='sign-in-container'>
```

Comment:
I see camelCase class names above and now snake-case.. always stick to 1 naming convention

Reply:
![Screenshot (140)](https://user-images.githubusercontent.com/73981407/221431229-44c833af-34fc-4b73-9578-9f3e65688bd0.png)